bootstrap/secret/secret.go

```diff
@@ -76,7 +76,7 @@ func NewSecretProvider(
 					lc.Debugf("SecretsFile is '%s'", secretConfig.SecretsFile)
 
 					if len(strings.TrimSpace(secretConfig.SecretsFile)) > 0 {
-						err = secureProvider.LoadServiceSecrets(secretConfig.SecretsFile)
+						err = secureProvider.LoadServiceSecrets(secretStoreConfig.SecretsFile, secretStoreConfig.DisableScrubSecretsFile)
```

Comment:
Now we have two params from the same instance of  `secretStoreConfig`.  would like to suggest refactoring this to just pass in `secretStoreConfig` instance so that the logic of dependency is hidden behind that func and also easier to extend later on if there are more.

Reply:
Sure

Reply:
Done